backend/layers/business/business.py

```diff
@@ -692,6 +693,20 @@ def tombstone_collection(self, collection_id: CollectionId) -> None:
         self.delete_all_dataset_versions_from_public_bucket_for_collection(collection_id)
         self.database_provider.tombstone_collection(collection_id)
 
+    def resurrect_collection(self, collection_id: CollectionId) -> None:
+        """
+        Resurrect a tombstoned Collection (untombstone Collection and un-delete public s3 assets). For admin use only.
```

Comment:
Initially reading this it was easy to assume this would only restore a single collection version and it's datasets. The code looks like it restores all previous version of the collection and previous versions of the datasets. Include that in the description or change the code.